source/frontend/src/features/leases/detail/LeasePages/insurance/details/Insurance.tsx

```diff
@@ -31,18 +31,18 @@ const InsuranceDetailsView: React.FunctionComponent<
     <div data-testid="insurance-section">
       <Section header="Required insurance">
         <InsuranceTypeList>
-          {sortedInsuranceList.map((insurance: Api_Insurance, index: number) => (
+          {sortedInsuranceList.map((insurance: ApiGen_Concepts_Insurance, index: number) => (
             <li key={`insurance-section-${insurance?.id?.toString() ?? index}`}>
-              {insurance.insuranceType.description}
-              {insurance.insuranceType.id === 'OTHER' && insurance.otherInsuranceType
+              {insurance.insuranceType?.description}
```

Comment:
interesting that this is coming from the generated models? as far as I can tell this is a required, non-nullable field in the DB.

Reply:
I think the problem is that string is still an object on the backend and that could be nullable according to our nullable checks